microsetta_admin/templates/create_kits.html

```diff
@@ -80,6 +108,17 @@ <h3>Microsetta Create Kits</h3>
         <script>
            document.agForm.num_kits.focus()
         </script>
+
```

Comment:
Some form adjustments to make this more clear visually:
- Default Samples per Kit to a value of 1
- For each sample per kit, have a clearly delineated section below the "Projects" dropdown that allows the user to either upload a CSV file or barcodes or click a checkbox to Generate Barcodes. There should not be any text inputs for barcodes.
- Move the "Create kits" button to the bottom of the form, under all of the sections for upload CSV/Generate Barcodes.